### DISCUSSION TITLE
Undefined method, Laravel 10

### DISCUSSION THREAD

**Discussion by gabyyro** [spatie/laravel-notification-log#14] [Q&A]
Hi, I installed and ran migration on Laravel 10 project. When I use $user->loggedNotifications() I get:

`Call to undefined method App\Models\User::loggedNotifications()`

I do not see this method inside package and I did not find what to import in documentation.

**Comment by moferry96** [comment]
Hi
Did you import Spatie\NotificationLog\Models\Concerns\HasNotifiableHistory trait into your User model?

you can find corresponding docs [here](https://spatie.be/docs/laravel-notification-log/v1/basic-usage/querying-the-notification-log#:~:text=USING%20THE%20NOTIFIABLE%20TO%20SEARCH%20LOGGED%20NOTIFICATIONS)

**Comment by gabyyro** [comment]
Yes, I reinstalled latest version and it now works with $user->notificationLogItems(), not with $user->loggedNotifications(). 

Thank you for answer.
